src/kibana/directives/in_time_errors.js

```diff
@@ -0,0 +1,30 @@
+define(function (require) {
+  var module = require('modules').get('kibana');
+  var $ = require('jquery');
+
+  module.directive('inTimeErrors', function () {
+    return {
+      restrict: 'C',
+      link: function ($scope, $elem, attrs) {
+        // this directive requires a Form element, with a name to reference the errors
+        if (!$elem.is('form') || !attrs.name) {
+          // TODO maybe throw an error here?
```

Comment:
Ugh. I know!
The `ng-form` directive doesn't have a controller, and when you try and require a directive without a controller bad things happen.